### ISSUE TITLE
Pagination component - update css used to hide elements

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#1470]
The Pagination component currently includes the following element, which only displays in smaller views:
`<div class="pf-c-pagination__total-items">20 items</div>`

Currently when viewing the full size version of the component, this element is hidden using `display: none;`. This property alone doesn't reliably hide elements from screen readers. Whenever hiding elements from screen readers using only css, the following properties should be defined:
```
display: none;
visibility: hidden;
```

These properties should also be applied to the elements that become hidden in the responsive view of this component.
